docs/setup/glue.md

```diff
@@ -67,8 +67,9 @@ If you are using the example notebook from glue, the first cell should now look
 # Sedona Config
 %extra_jars https://repo1.maven.org/maven2/org/apache/sedona/sedona-spark-shaded-3.3_2.12/{{ sedona.current_version }}/sedona-spark-shaded-3.3_2.12-{{ sedona.current_version }}.jar, https://repo1.maven.org/maven2/org/datasyslab/geotools-wrapper/{{ sedona.current_geotools }}/geotools-wrapper-{{ sedona.current_geotools }}.jar
 %additional_python_modules apache-sedona=={{ sedona.current_version }}
+```
```

Comment:
```suggestion
```

Reply:
https://github.com/apache/sedona/blob/master/docs/setup/glue.md#notebook-job

See the original